src/payment/requestor.rs

```diff
@@ -49,8 +49,13 @@ impl PaymentRequestorApi {
         Self { client, config }
     }
 
-    pub async fn get_debit_notes(&self) -> Result<Vec<DebitNote>> {
-        self.client.get("requestor/debitNotes").send().json().await
+    pub async fn get_debit_notes(&self, afterTimestamp: Option<DateTime<Utc>>, maxItems: Option<i64>) -> Result<Vec<DebitNote>> {
```

Comment:
I'd rather go with `u64` for `max_items` since negative values do not make sense here.

Reply:
Also: why not make it a template method like `get_invoices`?

Reply:
1) i started with a `u32` since this was the data-type that made most sense to me. But Diesel's `query.limit()` takes an `i64`
2) was only updating invoice_flow for the test scenario, good catch

Reply:
We shouldn't allow implementation details (Diesel interface) affect our client interface. Let's go with `u32` then.